### PR TITLE
HW04 Robin Oh

### PR DESCRIPTION
**Pull request by OhRobin**
1. I couldn't finish some of the problems in time.
2. One program didn't run on my terminal for some reason

### CODE REVIEW

File: HW04_ch07_ex02.py [hunk 1]
```diff
@@ -21,15 +21,24 @@
 
 ###############################################################################
 # Imports
+import math
 
 
 # Body
-
+def eval_loop():
```
Comment (danielsgriffin) [inline HW04_ch07_ex02.py]: What if 'done` is the first input?

File: HW04_ch08_ex00.py [hunk 2]
```diff
@@ -21,14 +21,23 @@
 
 # Body
 
+def count(inputSTring, inputLetter):
+	count = 0
+	for letter in inputSTring:
+		if letter == inputLetter:
+			count = count + 1
```
Comment (danielsgriffin) [inline HW04_ch08_ex00.py]: count += 1

File: HW04_ch08_ex00.py [hunk 3]
```diff
@@ -21,14 +21,23 @@
 
 # Body
 
+def count(inputSTring, inputLetter):
```
Comment (danielsgriffin) [inline HW04_ch08_ex00.py]: inputSTring = very unconventional for python
see: https://www.python.org/dev/peps/pep-0008/

File: HW04_string2.py [hunk 4]
```diff
@@ -16,9 +16,17 @@
 # add 'ly' instead.
 # If the string length is less than 3, leave it unchanged.
 # Return the resulting string.
+
+
+#THIS FILE WON'T RUN VIA THE GITBASH COMMAND LINE
```
Comment (danielsgriffin) [inline HW04_string2.py]: Please show me this.

**Comment by danielsgriffin** [comment]
Good job. Please resubmit once complete.

File: HW04_string2.py [hunk 5]
```diff
@@ -30,8 +38,19 @@ def verbing(s):
 # So 'This dinner is not that bad!' yields:
 # This dinner is good!
 def not_bad(s):
-    # +++your code here+++
-    return
+    x = s.split()
+
+    fromNum
+    toNum
+    for number in len(x)
```
Comment (Nic-Mon) [inline HW04_string2.py]: Forgot colon after for loop

`for number in len(x):`

File: HW04_string2.py [hunk 6]
```diff
@@ -30,8 +38,19 @@ def verbing(s):
 # So 'This dinner is not that bad!' yields:
 # This dinner is good!
 def not_bad(s):
-    # +++your code here+++
-    return
+    x = s.split()
+
+    fromNum
```
Comment (Nic-Mon) [inline HW04_string2.py]: giving error about referencing fromNum variable before assigning it.  Maybe use the .find('not') and .find('bad') methods to find the indexes for those words

File: D04_ex00.py [hunk 7]
```diff
@@ -11,19 +11,32 @@
...
+				return
+			elif int(guessNum) > x:
+				print("you've guessed too high")
+			else:
+				print("you've guessed too low")
 
 
 ################################################################################
 def main():
+	alphago();
```
Comment (jollylst) [inline D04_ex00.py]: what if I enter "Hello World" ?

File: HW04_ch07_ex02.py [hunk 8]
```diff
@@ -21,15 +21,24 @@
...
+		else:
+			y = eval(x)
+			print(y)
 
 ###############################################################################
 def main():
-    pass  # Remove this line and uncomment below once eval_loop is defined.
-    # eval_loop()
+    # Remove this line and uncomment below once eval_loop is defined.
+    eval_loop()
```
Comment (jollylst) [inline HW04_ch07_ex02.py]: what if input is "string"?

**Pull request closed without merging**